src/pybind/mgr/orchestrator.py

```diff
@@ -458,13 +492,13 @@ def blink_device_light(self, ident_fault, on, locations):
         """
         raise NotImplementedError()
 
-    def update_mgrs(self, num, hosts):
+    def update_mgrs(self, num, hosts, names):
         # type: (int, List[str]) -> WriteCompletion
```

Comment:
```suggestion
         # type: (int, List[str], List[str]) -> WriteCompletion
```

The parameter `name` needs to be added to all deriving orchestrators.

Reply:
oops that hunk doesn't belong, dropped